forester/src/rollover/operations.rs

```diff
@@ -68,14 +69,24 @@ pub async fn is_tree_ready_for_rollover<R: RpcConnection>(
             let threshold = ((1 << height) * account.metadata.rollover_metadata.rollover_threshold
                 / 100) as usize;
 
+            //  TODO: (fix) check to avoid processing Merkle trees with rollover threshold 0 which haven't processed any transactions
+            // let lamports_in_account_are_sufficient_for_rollover = account_info.lamports
+            //     > account.metadata.rollover_metadata.rollover_fee * (1 << height);
             Ok(merkle_tree.next_index() >= threshold)
         }
         TreeType::Address => {
             let account = rpc
                 .get_anchor_account::<AddressMerkleTreeAccount>(&tree_pubkey)
                 .await?
                 .unwrap();
-            info!("Account: {:?}", account);
+            let queue_account = rpc
+                .get_anchor_account::<QueueAccount>(&account.metadata.associated_queue)
+                .await?
+                .unwrap();
+            // let account_info = rpc
+            //     .get_account(account.metadata.associated_queue)
+            //     .await?
+            //     .unwrap();
```

Comment:
can we remove it?

Reply:
I left it on purpose so that we know that we need to take the queue account info not the Merkle tree.
This is different for state and address Merkle trees because for address Merkle trees users only interact with the queue the tree is only updated by the forester.